### PR TITLE
rename mail and change order

### PR DESCRIPTION
**Pull request by Kaushik-Iyer**
This PR closes/references issue #11 

## How has this been tested?
Results on running the server locally are:

image

## Checklist
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation.
- [ ] My change is listed in the ``doc/changelog.rst``.

### CODE REVIEW

**Comment by Kaushik-Iyer** [comment]
@mariobehling I have deployed and added a screenshot above. Please have a look, thank you

File: src/pretalx/locale/ca/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -3964,15 +3964,15 @@ msgid "Outbox"
 msgstr ""
 
 #: pretalx/orga/templates/orga/base.html:385
-msgid "Templates"
+msgid "Compose"
 msgstr ""
```
Comment (marcoag) [inline src/pretalx/locale/ca/LC_MESSAGES/django.po:3968]:
```suggestion
msgstr "Compose"
```

File: src/pretalx/locale/ca/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -3964,15 +3964,15 @@ msgid "Outbox"
...
 
 #: pretalx/orga/templates/orga/base.html:388
-msgid "Compose E-mails"
+msgid "Sent"
 msgstr ""
```
Comment (marcoag) [inline src/pretalx/locale/ca/LC_MESSAGES/django.po:3972]:
```suggestion
msgstr "Sent"
```

File: src/pretalx/locale/ca/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -3964,15 +3964,15 @@ msgid "Outbox"
...
 
 #: pretalx/orga/templates/orga/base.html:391
-msgid "Sent E-mails"
+msgid "Templates"
 msgstr ""
```
Comment (marcoag) [inline src/pretalx/locale/ca/LC_MESSAGES/django.po:3976]:
```suggestion
msgstr "Templates"
```

File: src/pretalx/locale/es_MX/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -4208,15 +4208,15 @@ msgid "Outbox"
 msgstr ""
 
 #: pretalx/orga/templates/orga/base.html:385
-msgid "Templates"
+msgid "Compose"
 msgstr ""
```
Comment (marcoag) [inline src/pretalx/locale/es_MX/LC_MESSAGES/django.po:4212]:
```suggestion
msgstr "Redactar"
```

File: src/pretalx/locale/es_MX/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -4208,15 +4208,15 @@ msgid "Outbox"
...
 
 #: pretalx/orga/templates/orga/base.html:388
-msgid "Compose E-mails"
+msgid "Sent"
 msgstr ""
```
Comment (marcoag) [inline src/pretalx/locale/es_MX/LC_MESSAGES/django.po:4216]:
```suggestion
msgstr "Enviar"
```

File: src/pretalx/locale/es_MX/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -4208,15 +4208,15 @@ msgid "Outbox"
...
 
 #: pretalx/orga/templates/orga/base.html:391
-msgid "Sent E-mails"
+msgid "Templates"
 msgstr ""
```
Comment (marcoag) [inline src/pretalx/locale/es_MX/LC_MESSAGES/django.po:4220]:
```suggestion
msgstr "Plantillas"
```

File: src/pretalx/locale/id/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -3930,15 +3930,15 @@ msgid "Outbox"
...
 
 #: pretalx/orga/templates/orga/base.html:385
-msgid "Templates"
+msgid "Compose"
 msgstr ""
 
 #: pretalx/orga/templates/orga/base.html:388
-msgid "Compose E-mails"
+msgid "Sent"
 msgstr ""
```
Comment (marcoag) [inline src/pretalx/locale/id/LC_MESSAGES/django.po:3938]:
```suggestion
msgstr "Kirim"
```

File: src/pretalx/locale/id/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -3930,15 +3930,15 @@ msgid "Outbox"
...
 
 #: pretalx/orga/templates/orga/base.html:385
-msgid "Templates"
+msgid "Compose"
 msgstr ""
```
Comment (marcoag) [inline src/pretalx/locale/id/LC_MESSAGES/django.po:3934]:
```suggestion
msgstr "Menyusun"
```

File: src/pretalx/locale/id/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -3930,15 +3930,15 @@ msgid "Outbox"
...
 
 #: pretalx/orga/templates/orga/base.html:388
-msgid "Compose E-mails"
+msgid "Sent"
 msgstr ""
 
 #: pretalx/orga/templates/orga/base.html:391
-msgid "Sent E-mails"
+msgid "Templates"
 msgstr ""
```
Comment (marcoag) [inline src/pretalx/locale/id/LC_MESSAGES/django.po:3942]:
```suggestion
msgstr "Templats"
```

File: src/pretalx/locale/pt_BR/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -4402,16 +4402,16 @@ msgid "Outbox"
...
-msgid "Templates"
-msgstr "Modelos"
+msgid "Compose"
+msgstr "Escrever emails"
 
 #: pretalx/orga/templates/orga/base.html:388
-msgid "Compose E-mails"
-msgstr "Escrever emails"
+msgid "Sent"
+msgstr "emails enviados"
```
Comment (marcoag) [inline src/pretalx/locale/pt_BR/LC_MESSAGES/django.po:4410]:
```suggestion
msgstr "Emails enviados"
```

File: src/pretalx/locale/cs/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -4377,16 +4377,16 @@ msgid "Outbox"
 msgstr "K odeslání"
 
 #: pretalx/orga/templates/orga/base.html:385
-msgid "Templates"
-msgstr "Šablony"
+msgid "Compose"
+msgstr "Poslání e-mailu"
```
Comment (marcoag) [inline src/pretalx/locale/cs/LC_MESSAGES/django.po:4381]: Let's just have consistency and use "email":
```suggestion
msgstr "Poslání emailu"
```

File: src/pretalx/locale/cs/LC_MESSAGES/django.po [hunk 12]
```diff
@@ -4377,16 +4377,16 @@ msgid "Outbox"
...
 
 #: pretalx/orga/templates/orga/base.html:388
-msgid "Compose E-mails"
-msgstr "Poslání e-mailu"
+msgid "Sent"
+msgstr "Odeslané e-maily"
```
Comment (marcoag) [inline src/pretalx/locale/cs/LC_MESSAGES/django.po:4385]:
```suggestion
msgstr "Odeslané emaily"
```

File: src/pretalx/locale/de_DE/LC_MESSAGES/django.po [hunk 13]
```diff
@@ -4458,16 +4458,16 @@ msgid "Outbox"
...
-msgid "Templates"
-msgstr "Vorlagen"
+msgid "Compose"
+msgstr "E-Mails schreiben"
 
 #: pretalx/orga/templates/orga/base.html:388
-msgid "Compose E-mails"
-msgstr "E-Mails schreiben"
+msgid "Sent"
+msgstr "Verschickte E-Mails"
```
Comment (marcoag) [inline src/pretalx/locale/de_DE/LC_MESSAGES/django.po:4466]:
```suggestion
msgstr "Verschickte emails"
```

File: src/pretalx/locale/de_Formal/LC_MESSAGES/django.po [hunk 14]
```diff
@@ -4484,16 +4484,16 @@ msgid "Outbox"
 msgstr "Postausgang"
 
 #: pretalx/orga/templates/orga/base.html:385
-msgid "Templates"
-msgstr "Vorlagen"
+msgid "Compose"
+msgstr "E-Mails schreiben"
```
Comment (marcoag) [inline src/pretalx/locale/de_Formal/LC_MESSAGES/django.po:4488]:
```suggestion
msgstr "Emails schreiben"
```

File: src/pretalx/locale/de_Formal/LC_MESSAGES/django.po [hunk 15]
```diff
@@ -4484,16 +4484,16 @@ msgid "Outbox"
...
 
 #: pretalx/orga/templates/orga/base.html:388
-msgid "Compose E-mails"
-msgstr "E-Mails schreiben"
+msgid "Sent"
+msgstr "Verschickte E-Mails"
```
Comment (marcoag) [inline src/pretalx/locale/de_Formal/LC_MESSAGES/django.po:4492]:
```suggestion
msgstr "Verschickte emails"
```

File: src/pretalx/locale/el/LC_MESSAGES/django.po [hunk 16]
```diff
@@ -4484,16 +4484,16 @@ msgid "Outbox"
 msgstr "Εξερχόμενα"
 
 #: pretalx/orga/templates/orga/base.html:385
-msgid "Templates"
-msgstr "Πρότυπο"
+msgid "Compose"\
+msgstr "Σύνταξη e-mail"
```
Comment (marcoag) [inline src/pretalx/locale/el/LC_MESSAGES/django.po:4488]:
```suggestion
msgstr "Σύνταξη email"
```

File: src/pretalx/locale/nl/LC_MESSAGES/django.po [hunk 17]
```diff
@@ -4167,16 +4167,16 @@ msgid "Outbox"
 msgstr "Uitgaand"
 
 #: pretalx/orga/templates/orga/base.html:385
-msgid "Templates"
-msgstr "Sjablonen"
+msgid "Compose"
+msgstr "Schrijf E-mails"
```
Comment (marcoag) [inline src/pretalx/locale/nl/LC_MESSAGES/django.po:4171]:
```suggestion
msgstr "Schrijf emails"
```

File: src/pretalx/locale/nl/LC_MESSAGES/django.po [hunk 18]
```diff
@@ -4167,16 +4167,16 @@ msgid "Outbox"
...
 
 #: pretalx/orga/templates/orga/base.html:388
-msgid "Compose E-mails"
-msgstr "Schrijf E-mails"
+msgid "Sent"
+msgstr "Verstuur E-mails"
```
Comment (marcoag) [inline src/pretalx/locale/nl/LC_MESSAGES/django.po:4175]:
```suggestion
msgstr "Verstuur emails"
```

File: src/pretalx/locale/pt_PT/LC_MESSAGES/django.po [hunk 19]
```diff
@@ -4449,16 +4449,16 @@ msgid "Outbox"
 msgstr "Caixa de Saída"
 
 #: pretalx/orga/templates/orga/base.html:385
-msgid "Templates"
-msgstr "Modelos"
+msgid "Compose"
+msgstr "Compor e-mails"
```
Comment (marcoag) [inline src/pretalx/locale/pt_PT/LC_MESSAGES/django.po:4453]:
```suggestion
msgstr "Compor emails"
```

File: src/pretalx/locale/pt_PT/LC_MESSAGES/django.po [hunk 20]
```diff
@@ -4449,16 +4449,16 @@ msgid "Outbox"
...
 
 #: pretalx/orga/templates/orga/base.html:388
-msgid "Compose E-mails"
-msgstr "Compor e-mails"
+msgid "Sent"
+msgstr "E-mails Enviados"
```
Comment (marcoag) [inline src/pretalx/locale/pt_PT/LC_MESSAGES/django.po:4457]:
```suggestion
msgstr "Emails enviados"
```

**Review by marcoag — CHANGES_REQUESTED** [review]

**Comment by Kaushik-Iyer** [comment]
@mariobehling  @marcoag I have committed all the suggestions. But there are merge conflicts. I think the code lines mentioned in the files have been updated. You might want to have a look at one of the conflict files. Should I solve the merge conflicts also?

**Comment by norbusan** [comment]
Yes, please merge the `main` branch into your branch, that is the normal procedure. Thanks

**Comment by Kaushik-Iyer** [comment]
@norbusan I've tried to merge changes but this PR now looks really ugly. I'm assuming I need to create a new PR for the development branch instead of the main branch, or did I do something wrong?

**Comment by norbusan** [comment]
Yes please, since and main and development are more or less the same, I suggest re-opening a new PR against development from the same branch. If you do this, please close this PR.

**Comment by Kaushik-Iyer** [comment]
Re-opened new PR against development branch. Closing this PR now

**Pull request closed without merging**